### DISCUSSION TITLE
连续调用两次相同的接口,chrome nextwork只有一次,但是alova会有两次返回值,请问是什么原因呢

### DISCUSSION THREAD

**Discussion by bailihuiyue** [alovajs/alova#247] [Q&A]
### 请详细描述bug

例如:两个不同的接口
![1701660548584](https://github.com/alovajs/alova/assets/33573384/864aeafd-101b-4534-84d1-5daa2a221968)
![image](https://github.com/alovajs/alova/assets/33573384/f6c93bcd-9261-4274-b8ed-e947d809f357)
l两个相同的接口
![image](https://github.com/alovajs/alova/assets/33573384/2aa8b41c-a675-4b03-9621-63a66be9f0e4)
![image](https://github.com/alovajs/alova/assets/33573384/55768e40-b1c2-428c-a794-32f411331347)
请问如何解决

### 复现步骤

例如:两个不同的接口
![1701660548584](https://github.com/alovajs/alova/assets/33573384/864aeafd-101b-4534-84d1-5daa2a221968)
![image](https://github.com/alovajs/alova/assets/33573384/f6c93bcd-9261-4274-b8ed-e947d809f357)
l两个相同的接口
![image](https://github.com/alovajs/alova/assets/33573384/2aa8b41c-a675-4b03-9621-63a66be9f0e4)
![image](https://github.com/alovajs/alova/assets/33573384/55768e40-b1c2-428c-a794-32f411331347)
请问如何解决

### alova 版本

2.9.1

**Comment by JOU-amjs** [comment]
> 问题类内容请在discussion提问

这是因为alova默认是开启了[请求共享](https://alova.js.org/tutorial/next-step/share-request/)的
